### PR TITLE
Py36 testing

### PR DESCRIPTION
**Pull request by cxhernandez**
moving #989 to here

re: #988

### CODE REVIEW

**Comment by cxhernandez** [comment]
@mpharrigan: should we build the docs in 3.6 now?

**Comment by mpharrigan** [comment]
> @mpharrigan: should we build the docs in 3.6 now?

sure

Why did you make a new PR?

Also, we could keep the py3.5 on numpy 1.11 to test against multiple versions of numpy. Our three configurations would be loosely: a python2 user, a slow-to-upgrade python3 user, a bleeding-edge user

**Comment by cxhernandez** [comment]
> Why did you make a new PR?

GitHub didn't let me push to this repo remotely

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 numpydoc
 matplotlib
-ipython-notebook
+jupyter
+notebook
```
Comment (mpharrigan) [inline docs/requirements.txt]: is this right? notebook?

Comment (cxhernandez) [inline docs/requirements.txt]: https://pypi.python.org/pypi/notebook

**Pull request merged**